packages/nocodb/src/models/Column.ts

```
@@ -857,6 +883,48 @@ export default class Column<T = any> implements ColumnType {
+      for (const aiCol of aiColumns) {
+        const ai = await new Column(aiCol).getColOptions<AIColumn>(
+          context,
+          ncMeta,
+        );
+
+        if (!ai) continue;
+
+        /*
+          if prompt includes deleted column id {column_id}, add error and update
+        */
+        if (ai.prompt && ai.prompt.match(/{column_id}/)) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)